src/voxel.h

```diff
@@ -145,9 +140,9 @@ class VoxelArea
 
 	s32 getVolume() const
 	{
-		v3s16 e = getExtent();
-		return (s32)e.X * (s32)e.Y * (s32)e.Z;
+		return (s32) m_cache_extent.X * (s32) m_cache_extent.Y * (s32) m_cache_extent.Z;
```

Comment:
if i remember we don't have a guideline for this, i prefer the space for readability